<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettings</class>
 <widget class="QDialog" name="DialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="7" column="1">
          <widget class="QLabel" name="labelNob">
           <property name="text">
            <string>nob (number of blocks)</string>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QComboBox" name="comboBoxTheme"/>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="checkBoxBoard2">
           <property name="text">
            <string>board 2</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxNob">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>4294967295.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxNos">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>2.000000000000000</double>
           </property>
           <property name="maximum">
            <double>4294967295.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="10" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="labelNos">
           <property name="text">
            <string>nos (number of samples)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelBoardSel">
           <property name="text">
            <string>PCIe Board select</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxContiniousPause_in_ms">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="suffix">
            <string>ms</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>4294967.294999999925494</double>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="labelContiniousPause">
           <property name="text">
            <string>continious pause</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="checkBoxBoard0">
           <property name="text">
            <string>board 0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="checkBoxBoard1">
           <property name="text">
            <string>board 1</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QCheckBox" name="checkBoxBoard4">
           <property name="text">
            <string>board 4 </string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>theme</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QCheckBox" name="checkBoxBoard3">
           <property name="text">
            <string>board 3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="camerasettings">
      <attribute name="title">
       <string>Camera Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="cameraSettingsTabs">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="board0">
          <attribute name="title">
           <string>Board 0</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="CameraSettingsWidget" name="cameraSettingsWidgetBoard0"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="board1">
          <attribute name="title">
           <string>Board 1</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="CameraSettingsWidget" name="cameraSettingsWidgetBoard1"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="board2">
          <attribute name="title">
           <string>Board 2</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="CameraSettingsWidget" name="cameraSettingsWidgetBoard2"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="board3">
          <attribute name="title">
           <string>Board 3</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="CameraSettingsWidget" name="cameraSettingsWidgetBoard3"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="board4">
          <attribute name="title">
           <string>Board 4</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="CameraSettingsWidget" name="cameraSettingsWidgetBoard4"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="pushButtonDefault">
       <property name="text">
        <string>Factory reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSettingsLevel">
       <item>
        <property name="text">
         <string>guided</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>free</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CameraSettingsWidget</class>
   <extends>QTabWidget</extends>
   <header>camerasettingswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>checkBoxBoard0</tabstop>
  <tabstop>checkBoxBoard1</tabstop>
  <tabstop>checkBoxBoard2</tabstop>
  <tabstop>checkBoxBoard3</tabstop>
  <tabstop>checkBoxBoard4</tabstop>
  <tabstop>doubleSpinBoxNos</tabstop>
  <tabstop>doubleSpinBoxNob</tabstop>
  <tabstop>doubleSpinBoxContiniousPause_in_ms</tabstop>
  <tabstop>comboBoxTheme</tabstop>
  <tabstop>pushButtonDefault</tabstop>
  <tabstop>comboBoxSettingsLevel</tabstop>
  <tabstop>cameraSettingsTabs</tabstop>
  <tabstop>cameraSettingsWidgetBoard0</tabstop>
  <tabstop>cameraSettingsWidgetBoard1</tabstop>
  <tabstop>cameraSettingsWidgetBoard2</tabstop>
  <tabstop>cameraSettingsWidgetBoard3</tabstop>
  <tabstop>cameraSettingsWidgetBoard4</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
